file-updates/manual-release.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Release
+metadata:
+  name: file-updates-test-manual-release
+  namespace: dev-release-team-tenant
+spec:
+  snapshot: fileupdatestest-kncqg
```

Comment:
little confused by this. How do we know this exact snapshot will exist? Do snapshots not use `generateName`?

Reply:
removed